mep/people/templates/people/borrowing_activities.html

```
@@ -0,0 +1,95 @@
+        <span>cards</span>
+    </li>
+</nav>
+
+<nav class="tabs subsection" aria-label="Activities">
+    <li class="tab" aria-selected="true">
+        <a href="{% url 'people:borrowing-activities' slug=member.slug %}">Borrowing</a>
+    </li>
+    <li class="tab">
+        <a href="{% url 'people:membership-activities' slug=member.slug %}">Membership</a>
+    </li>
+</nav>
+
+<div class="activities">
+    {% comment %} preliminary form controls for sorting and expand/collapse on mobile
+    <form id="activity-mobile-control">
+        <label class="sort">
+            <span>SORTED BY</span>
+            <select>
+                <option value="1">Type</option>
+                <option value="2">Category</option>
+                <option value="3">Duration</option>
+                <option value="4" selected="selected">Start Date</option>
+                <option value="5">End Date</option>
+                <option value="6">Amount</option>
+            </select>
+            <img class="dropdown icon" src="{% static 'img/icons/chevron_down.png' %}" alt="">
+        </label>
+        <button id="expand">Expand all</button>
+    </form>
+    {% endcomment %}
+
+    {% if not event_list.count %}
+    <p>No documented borrowing activity.</p>
+    {% else %}
+    <div class="wrapper">
+    <table id="activity-table">
+        <thead>
+            <tr>
+                <th>Title</th>
+                <th>Author</th>
+                <th>Publication Date</th>
+                <th>Status</th>
+                <th data-sort-default>Start Date</th>
+                <th>End Date</th>
+            </tr>
+        </thead>
+        {% for event in event_list %}
+        <tr>
+            <td class="title">{{ event.work.title }}</td>
+            <td class="author">{{ event.work.author_list }}</td>
+            <td class="pubdate">{{ event.work.year }}</td>
+            <td class="status">
+                {% if event.borrow %}{{ event.borrow.status }}
```

would `firstof` work here?

---

how would that work when testing for a property?

---

Might work, because django templates swallow errors... not sure.

However, based on the work I did with the mobile view, I think this should just be `Borrowed` or `Purchased` - the actual borrow event status is confusing in some cases.